src/services/clinical/index.js

```diff
@@ -152,7 +153,29 @@ const getClinicalData = async (variables, Authorization) => {
 
   const query = new URLSearchParams(filters).toString();
 
-  const url = `${CLINICAL_SERVICE_ROOT}/clinical/program/${programShortName}/clinical-data?${query}`;
+  const url = urlJoin(
+    `${CLINICAL_SERVICE_ROOT}/clinical/program/${programShortName}`,
+    `/clinical-data?${query}`,
+  );
+  const response = await fetch(url, {
+    method: 'get',
+    headers: { Authorization },
+  })
+    .then(restErrorResponseHandler)
+    .then((response) => response.json());
+
+  return response;
+};
+
+const getClinicalSearchResults = async (variables, Authorization) => {
+  const { programShortName, filters } = variables;
+
+  const query = new URLSearchParams(filters).toString();
+
+  const url = urlJoin(
+    `${CLINICAL_SERVICE_ROOT}/clinical/program/${programShortName}`,
```

Comment:
please separate variables from strings in this url join

Reply:
Updated